var/spack/repos/builtin/packages/py-numpy/package.py

```diff
@@ -158,6 +155,18 @@ def build_args(self, spec, prefix):
 
         return args
 
+    def setup_environment(self, spack_env, run_env):
+        python_version = self.spec['python'].version.up_to(2)
+
+        include_path = join_path(
+            self.prefix.lib,
+            'python{0}'.format(python_version),
+            'site-packages',
+            'numpy/core/include')
+
+        run_env.prepend_path('C_INCLUDE_PATH', include_path)
+        run_env.prepend_path('CPLUS_INCLUDE_PATH', include_path)
```

Comment:
I've never seen `C_INCLUDE_PATH` or `CPLUS_INCLUDE_PATH` before, I thought the correct name was `CPATH`? Should we add these to `etc/spack/defaults/modules.yaml`?

Reply:
`C_INCLUDE_PATH` / `CPLUS_INCLUDE_PATH` are used in C- / C++-compilations only, whereas `CPATH` is used in both.

I hardly ever use `CPATH` (and had to [read up](https://gcc.gnu.org/onlinedocs/cpp/Environment-Variables.html) on it), but here it should be sufficient to set `CPATH` only.

Reply:
Hrm.. adding both `C_INCLUDE_PATH` and `CPLUS_INCLUDE_PATH` is exactly the same as adding `CPATH`, so I think `etc/spack/defaults/modules.yaml` is fine.

One could even argue that this change should only set `CPATH` and not `C_INCLUDE_PATH` and `CPLUS_INCLUDE_PATH` (which was suggested by the error message I encountered).

It will work either way! :)

Reply:
If you don't mind, I would like this package to only set `CPATH` just to be in line with `modules.yaml`.